routing/transit_world_graph.hpp

```diff
@@ -30,7 +33,8 @@ class TransitWorldGraph final : public WorldGraph
                    std::vector<SegmentEdge> & edges) override;
   Junction const & GetJunction(Segment const & segment, bool front) override;
   m2::PointD const & GetPoint(Segment const & segment, bool front) override;
-  RoadGeometry const & GetRoadGeometry(NumMwmId mwmId, uint32_t featureId) override;
+  bool IsOneWay(NumMwmId mwmId, uint32_t featureId) override;
```

Comment:
Let's add note that all transit features are oneway.

Reply:
done